src/index.html

```diff
@@ -4,10 +4,327 @@
+              Nice work, Certificates National
+              Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="third-section__testimonial-card testimonial-card">
+            <img
+              src="./images/people/2.jpg"
+              alt="Scarlett Hawkins"
+              class="testimonial-card__img"
+            >
+            <a href="https://www.linkedin.com/" class="testimonial-card__name">
+              <h3>Scarlett Hawkins</h3>
+            </a>
+            <h4 class="testimonial-card__position">
+              Senior Vice President
+            </h4>
+            <p>
+              My soul is illuminated by an unearthly joy,
+              as these beautiful spring morning, which
+              I enjoy with all my heart
+            </p>
+          </div>
+
+          <div class="third-section__testimonial-card testimonial-card">
+            <img
+              src="./images/people/3.jpg"
+              alt="Max Henry"
+              class="testimonial-card__img"
+            >
+            <a href="https://www.linkedin.com/" class="testimonial-card__name">
+              <h3>Max Henry</h3>
+            </a>
+            <h4 class="testimonial-card__position">
+              Chief Design Officer
+            </h4>
+            <p>
+              When my lovely valley of steam rises and
+              half-day sun is above an impermeable
+            </p>
+          </div>
+        </div>
+
+      </section>
+
+      <a class="anchor" id="contacts"></a>
+      <section class="fourth-section main-content__last-section">
+
+        <div class="fourth-section__heading">
+          <h2>Contact us</h2>
+          <div class="fourth-section__decor-icon"></div>
+        </div>
+
+        <div class="fourth-section__contacts">
+
+          <div class="fourth-section__contact-info contact-info">
+            <div>
+              <h4 class="contact-info__heading">
+                Phone
+              </h4>
+              <a href="tel:+1 234 5555-55-55" class="contact-info__link">+1 234 5555-55-55</a>
+
+              <h4 class="contact-info__heading contact-info__heading--email">
+                Email
+              </h4>
+              <a href="mailto:hello@miami.com" class="contact-info__link">hello@miami.com</a>
+            </div>
+
+            <div>
+              <h4 class="contact-info__heading">
+                Address
+              </h4>
+              <a href="https://goo.gl/maps/FJ46Eh3aiQQ5VoTKA" class="contact-info__link">
+                <p class="contact-info__link-email">
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN
+                  <br>
+                  55401
+                </p>
+              </a>
+            </div>
+          </div>
+
+          <form
+            class="fourth-section__contact-form contact-form"
+            action="mailto:hello@miami.com"
```

Comment:
```suggestion
            action="#"
```
Let's make our form send data to the current page. In your case now it's insecure and gives you some troubles